web_src/less/_review.less

```diff
@@ -16,6 +16,13 @@
   }
 }
 
+.diff-file-box .lines-code .code-inner.has-escaped::before {
+  visibility: visible;
+  content: '⚠️';
```

Comment:
Reading code with legitimate bidirectional text is a little jarring with the warnings right next to it. Could it instead be put to the left of the line numbers? This is what GitHub has done, and I believe this is what done more commonly in editors (e.g. for syntax errors).